src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -142,7 +142,7 @@ run labels). When there is only one scan of a given type the run key MAY be
 omitted. Please note that diffusion imaging data is stored elsewhere (see
 below).
 
-The OPTIONAL `acq-<label>` key/value pair corresponds to a custom label the user
+The optional `acq-<label>` key/value pair corresponds to a custom label the user
```

Comment:
Any reason why lowercase `OPTIONAL`? It's supposed to be an indication that this is a RFC2119 label. If you are proposing to change that it should be done in another PR.

Reply:
ha, also found https://tools.ietf.org/html/rfc8174  ;-) so indeed then should be all UPPERCASE to be keywords.
Again -- I was just aiming for consistency and there were uses both ways, I will just push (later today) a fix to get them all UPPERCASED